packages/trees/frontend/sirius-components-trees/src/views/TreeView.tsx

```diff
@@ -233,28 +198,21 @@ export const TreeView = ({
   });
 
   return (
-    <>
-      <div data-testid={treeId}>
-        {tree ? (
-          <Tree
-            editingContextId={editingContextId}
-            tree={convertedTree}
-            onExpand={onExpand}
-            onExpandAll={onExpandAll}
-            readOnly={readOnly}
-            enableMultiSelection={enableMultiSelection}
-            markedItemIds={markedItemIds}
-            textToFilter={textToFilter}
-            textToHighlight={textToHighlight}
-            treeItemActionRender={treeItemActionRender}
-          />
-        ) : null}
-      </div>
-      <Toast
-        message={message}
-        open={toast === 'visible'}
-        onClose={() => dispatch({ type: 'HIDE_TOAST' } as HideToastEvent)}
-      />
-    </>
+    <div data-testid={treeId}>
+      {tree ? (
```

Comment:
- Do not use this way to express conditional rendering
- Why does tree can be `null` when it is explicitly described as non null in its type?